apps/nextjs-website/react-components/components/Header/Header.tsx

```diff
@@ -196,7 +195,7 @@ const Header = ({
           />
         )}
 
-        {!isMobile && drawer && (
+        {drawer && (
```

Comment:
Why was this removed?

Shouldn't the DesktopDrawer be rendered only on Desktop?